### DISCUSSION TITLE
Disable limit for "implausible width"

### DISCUSSION THREAD

**Discussion by joshinils** [streetcomplete/StreetComplete#4742] [Feedback]
Most of the residential roads here are no wider than 5.5m to 6m at most. Some even less than 3m.

What is the limit for this notice?

I find it annoying to ignore it every time i add a width measurement.

I am ok with the oneway quest afterward, that makes sense.
I would also like to know the threshold for that.

**Comment by Helium314** [comment]
> I would also like to know the threshold for that.

https://github.com/streetcomplete/StreetComplete/blob/46bea9bdf3a8a3174b563dc81e3213bbaaac8044/app/src/main/java/de/westnordost/streetcomplete/quests/oneway/AddOneway.kt#L28

**Comment by matkoniecz** [comment]
see #4655

https://github.com/streetcomplete/StreetComplete/commit/a36e2fb5942118a42027b35f06da127d1943ae3a

**Reply by matkoniecz** [reply]
If you have a specific idea how warning should be changed - please open an issue (I have no obvious ideas right now)

**Reply by Helium314** [reply]
The check seems to assume there are 2 full lanes, but often when there are no markings the width is somewhere between 1 and 2 lanes, especially outside cities.

**Reply by matkoniecz** [reply]
> The check seems to assume there are 2 full lanes

Are you sure?

See https://github.com/streetcomplete/StreetComplete/blob/8144401c26916044982d7b1ba2a435c3b5f308bf/app/src/main/java/de/westnordost/streetcomplete/osm/RoadWidth.kt#L16 - it seems that it should return 2.75 for `lanes=1` road 

Though it seems to not be present in tests - https://github.com/streetcomplete/StreetComplete/blob/e892b3c389bbccec5733c93631c763bc93a84aa7/app/src/test/java/de/westnordost/streetcomplete/osm/RoadWidthKtTest.kt

**Reply by Helium314** [reply]
Yes, only if there are no lane marking, which commonly is the case on such roads. SC wouldn't allow you to set 1 lane if it's not a one-way.

**Reply by Helium314** [reply]
I checked again, also using tests to confirm: roads eligible for the AddRoadWidth quest require width >= 5.2 m, unless they are one-way roads.
Even a 5 m wide non-oneway road tagged with `lanes=1` will trigger the dialog.

Roads that are already tagged as single lane should only require single lane width, and probably road types like residential or living street without lane markings should require something like 4 m. It's rather common that 2 trucks can't pass each other on such roads, at least outside cities.

**Comment by legofahrrad** [comment]
It could be similar to the dialog when you're not on-site. There should be a checkbox "Don't show again for this session".

**Comment by mnalis** [comment]
Hm, yeah, it happens to me quite often for smaller residential streets. e.g. https://www.openstreetmap.org/note/3519988 and https://www.openstreetmap.org/note/3519989, but there were many more. 

I've learned to live with it; because the check does makes sense - it is the road builders practice that seems problematic (building a two-way street that may not be used by cars going in both directions). Better to be warned about it.
